nautobot/core/views/renderers.py

```diff
@@ -233,6 +261,20 @@ def get_context(self, data, accepted_media_type, renderer_context):
                         "fields": view.bulk_create_form_class(model).fields if view.bulk_create_form_class else None,
                     }
                 )
+            elif view.action == "changelog":
+                context.update(
+                    {
+                        "base_template": get_base_template(data.get("base_template"), model),
+                        "active_tab": "changelog",
+                    }
+                )
+            elif view.action == "notes":
+                context.update(
+                    {
+                        "base_template": get_base_template(data.get("base_template"), model),
+                        "active_tab": "notes",
+                    }
+                )
```

Comment:
Since these are pretty much identical, couldn't we get away with this?
```suggestion
            elif view.action in ["changelog", "notes"]:
                context.update(
                    {
                        "base_template": get_base_template(data.get("base_template"), model),
                        "active_tab": view.action,
                    }
                )
```